Source/DataAccessLayer.SqlClient/SqlEntityProvider.cst

```diff
@@ -44,8 +44,9 @@ namespace <%=DALNameSpace%>.SqlClient
 		/// Uses connection string to connect to datasource.
 		/// </summary>
 		/// <param name="connectionString">The connection string to the database.</param>
+		/// <param name="defaultCommandTimeout">The command timeout for the database calls.</param>
 		/// <param name="useStoredProcedure">A boolean value that indicates if we use the stored procedures or embedded queries.</param>
 		/// <param name="providerInvariantName">Name of the invariant provider use by the DbProviderFactory.</param>
-		public <%=ProviderName%>(string connectionString, bool useStoredProcedure, string providerInvariantName): base(connectionString, useStoredProcedure, providerInvariantName){}
+		public <%=ProviderName%>(string connectionString, int defaultCommandTimeout, bool useStoredProcedure, string providerInvariantName): base(connectionString, defaultCommandTimeout, useStoredProcedure, providerInvariantName){}

```

Comment:
I think it would be best to move default command timeout to the last argument. That way committed generated code diff won't be that bad and you won't wonder what changed as the whole line will be highlighted.

Reply:
github must use really bad diff software if that happens.  I was just grouping it next to the connectionString as I felt the two belonged more together.  It would not hurt my feelings to move it though.